handlers/feedback.go

```diff
@@ -123,13 +124,13 @@ func getFeedback(w http.ResponseWriter, req *http.Request, url, errorType, descr
 }
 
 // AddFeedback handles a users feedback request and sends a message to slack
-func AddFeedback(to, from string, isPositive bool, rend interfaces.Renderer, emailSender email.Sender, cacheService *cacheHelper.Helper) http.HandlerFunc {
+func AddFeedback(to, from string, isPositive bool, rend interfaces.Renderer, cacheService *cacheHelper.Helper, feedbackCfg *config.FeedbackConfig) http.HandlerFunc {
```

Comment:
just a suggestion to simplify the signature:
As `from` and `to` values come from the config, maybe you could just pass a pointer to the full config:  `cfg *config.Config` (which will also contain the feedback config, once complete)

Reply:
Ended up removing from and to as they are no longer used.